Documentation/introduction.rst

```diff
@@ -22,13 +22,12 @@ some of the largest supercomputers worldwide in a pioneering simulation of the 1
 engineering frequency band (0-10 Hz) were modeled on a non-planar
 earthquake fault structure. In early 2017, SeisSol performed the longest
 and largest dynamic rupture scenario to date, enabled by local time
-stepping [Uphoff et al.,2017], resolving the 2004
+stepping (Uphoff et al.,2017), resolving the 2004
 Sumatra-Andaman earthquake including complex splay fault geometries. The
 paper won the prestigious “Best Paper Award” of the International
 Supercomputing Conference (SC17). SeisSol results imply that
-acknowledging geometrical complexity, realistic fault properties and
+acknowledging geometrical complexity, realistic fault properties, and
 velocity models affect not only earthquake source dynamics but the
 synthetic ground shaking crucially. The software package is available to
-the community as an open-source distribution
-(`www.github.com/SeisSol/SeisSol <www.github.com/SeisSol/SeisSol>`__).
```

Comment:
This is a clickable link. See here:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages
Please revert this change.

Reply:
the previous version lead to this broken link if we clicked it: https://github.com/SeisSol/SeisSol/blob/master/Documentation/www.github.com/SeisSol/SeisSol

Reply:
Here is an examples which works:
```
... using Gmsh are provided
at this `link <https://github.com/SeisSol/SeisSol/tree/master/preprocessing/meshing/gmsh_example>`_.
```

Reply:
> If the link text should be the web address, you don’t need special markup at all, the parser finds links and mail addresses in ordinary text.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

Reply:
Ok then, the change is fine!